src/css/components/Nav.scss

```diff
@@ -142,6 +149,15 @@
   display: block;
 }
 
+.Nav-caret {
+  width: scaleGrid(2);
+  display: block;
+}
+
+.Nav-sectionItems {
+  margin: 0 8px;
+}
+
```

Comment:
Super minor nit (sorry!): Can we indent these so that they're aligned to the appropriate scope?